model/vcs/metrics.yaml

```diff
@@ -43,18 +43,38 @@ groups:
         requirement_level: recommended
       - ref: vcs.ref.head.name
         requirement_level: required
+  - id: metric.vcs.change.time_to_merge
+    type: metric
+    metric_name: vcs.change.time_to_merge
+    brief: 'The amount of time since its creation it took a change (pull request/merge request/changelist) to get merged into the target(base) ref.'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.name
+        requirement_level: recommended
+      - ref: vcs.ref.head.name
+        requirement_level: required
+      - ref: vcs.ref.head.revision
+        requirement_level: recommended
+      - ref: vcs.ref.base.name
+        requirement_level: recommended
+      - ref: vcs.ref.base.revision
+        requirement_level: recommended
```

Comment:
Should `revision` be a metric attribute?
It would create a separate time series anytime the revision changes. Ie it could cause issues with cardinality.

Reply:
I set it to recommended instead of required. It's can be a useful attribute in context, but completely optional.